daemon/execdriver/driver.go

```diff
@@ -93,6 +93,11 @@ type Pid struct {
 	HostPid bool `json:"host_pid"`
 }
 
+// UTS settings of the container
+type Uts struct {
+	HostUts bool `json:"host_uts"`
```

Comment:
@ibuildthecloud I'm not sure about the usage either. Following what was done for ipc is sufficient.